src/ResourceManager/Compute/Commands.Compute/help/Update-AzureRmVmss.md

```diff
@@ -365,6 +382,7 @@ This setting affects the consistency and performance of the disk.
 Type: CachingTypes
 Parameter Sets: (All)
 Aliases: 
+Accepted values: None, ReadOnly, ReadWrite
```

Comment:
The description seems to be out of date - we could either update it or I can file a bug against VMSS.